server/hotrod/src/test/java/org/infinispan/server/hotrod/HotRodMergeTest.java

```diff
@@ -85,10 +84,10 @@ public void testNewTopologySentAfterCleanMerge(Method m) {
       TestingUtil.waitForNoRebalance(cache(p0.node(0)));
       expectPartialTopology(client, initialTopology + 1);
       partition(0).merge(partition(1));
-      eventuallyExpectCompleteTopology(client, initialTopology + 8);
+      eventuallyExpectCompleteTopology(client, initialTopology + 7);
```

Comment:
Why was the previous assumption `+ 8` and now it's `+ 7`?

Reply:
This is because of https://github.com/ryanemerson/infinispan/blob/a0425f55af69d1bdea5af5dd1766ffdb74b4b123/core/src/main/java/org/infinispan/partitionhandling/impl/PreferAvailabilityStrategy.java#L227